doc/release-process.md

```diff
@@ -166,10 +205,4 @@ navcoin.org (see below for navcoin.org update instructions).
 
 - Announce the release:
 
-- Communicate with enough time to economic actors of the Network (exchanges):
```

Comment:
why has this been removed?

Reply:
This has been added back in but GitHub is not showing it :)